transcom/react-file-viewer
# fix html loader for dev webpack

## Summary
Resolves the following error when serving dev. No version bump needed, this is for dev.

```
ERROR in   Error: The loader didn't return html.

  - index.js:323 HtmlWebpackPlugin.evaluateCompilationResult
    [react-file-viewer]/[html-webpack-plugin]/index.js:323:24

  - index.js:230
    [react-file-viewer]/[html-webpack-plugin]/index.js:230:22

  - task_queues:95 process.processTicksAndRejections
    node:internal/process/task_queues:95:5

  - async Promise.all

  - async Promise.all
```

## How to test
1. Clone locally
2. `yarn install`
3. `yarn serve:dev`
4. Verify http://localhost:8081/ loads a PDF
